frontend/whiteboard-app/src/components/WhiteboardPage.tsx

```diff
@@ -1,25 +1,80 @@
-import React from "react";
-import useNoteRequest from "../hooks/useNoteRequest";
+import React, { useState, ChangeEvent, FormEvent } from "react";
 
-const WhiteboardPage = () => {
-    const {notes} = useNoteRequest();
-    return  (
-        <>
-            <h1>Whiteboard</h1>
-            <ul>
-              {
-                notes !== undefined
-               ? (notes.map(note => (
-                <li key={note.noteId}>
-                  <h3>{note.title}</h3>
-                  <div>{note.description}</div>
-                </li>
-              )))
-              : (<div>Fehler beim Laden der Notizen</div>)
-               }
-            </ul>
-        </>
-    );
+interface Note {
```

Comment:
der typ/ das interface Note existiert schon in frontend/whiteboard-app/src/types.ts
deshalb musst du es hier nicht erneut deklarieren